app/uk/gov/hmrc/exports/services/mapping/CachingMappingHelper.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.services.mapping
+import uk.gov.hmrc.exports.models.declaration.ItemType.IdentificationTypeCodes.{
+  CUSCode,
+  CombinedNomenclatureCode,
+  NationalAdditionalCode,
+  TARICAdditionalCode
+}
+import uk.gov.hmrc.exports.models.declaration.{CommodityMeasure, ItemType}
+import uk.gov.hmrc.wco.dec._
+
+object CachingMappingHelper {
```

Comment:
It'd be nice if this was a class rather than an object so the tests of things that use it don't need to care about what it does

Reply:
Thsi has been addressed